### DISCUSSION TITLE
Discuss and determine book structure

### DISCUSSION THREAD

**Discussion by dskiles** [dskiles/eloransetting#2] [General]
Determine the high level structure of the book.

Acceptance criteria:

Accepted covers
* Overall theme and goal of book
* Chapter definitions

**Comment by dskiles** [comment]
Here's my suggestion
* The World
    * The Event
    * Eloran's Peculiar Moon and its Consequences
    * The continents of the known world
    * Using the world in your game
* The Imperium, or the Old Country
    * The holdings of the Imperium
    * Imperial teleportation
    * Imperial History
    * Imperial Culture
* Anem
    * A brief history of Anem
    * Anemese Geography
    * The government of Anem
        * Emperor
        * Military
    * Life in Anem
        * Culture
        * Religion
        * Food
        * Personal prospects (a day in the life)
     * Points of Interest
     * Using Anem in your game
* Pirra
    * A brief history of Pirra
    * Pirran Geography
    * The government of Pirra
        * Emperor
        * Military
    * Life in Pirra
        * Culture
        * Religion
        * Food
        * Personal prospects (a day in the life)
     * Points of Interest
         * Libussa
             * The Black Citadel
             * The Academy
             * Mad Pyotr's Tower
     * Using Pirra in your game
* REPEAT THE ABOVE FOR TEMEN, CHERNIK, AND JHILAL
* The Diaspora
    * A brief synopsis on what this was before The Event
    * A brief synopsis of what it is now
        * Geography
        * Biology
        * Residents
    * Points of interest
    * Using The Diaspora in your game
* The Moonshadow
    * A brief synopsis on what this was before The Event
    * A brief synopsis of what it is now
        * The Penumbra vs the Umbra
        * Geography
        * Biology
        * Residents
    * Points of interest
    * Using The Moonshadow in your game
* Notes on the Friends and Enemies of Man
    * Dwarves
        * Not meant to be playable, but we won't stop you
    * Elves
        * Not meant to be playable, but we won't stop you
    * Dark Elves
        * Not meant to be playable, and god help you if you try
    * Ancient Lizard Kingdoms
        * Use to explain how to build non-human dungeon experiences
    * Orcs
        * Intended to be playable.  Use orcs if your system has them, otherwise humans.
* Gods and Religion
    * Major pantheons
        * Major deities (imperial and shared)
        * Pirran Cycle
        * Anemese Cycle
        * Titans
        * Lesser Powers
* Beyond the World's Edge
    * Digression into Outside Context Problems
    * How to splice in other settings
    * Digression into the size of the continent, or:  you have plenty of room
    * The Second World to the West
    * The Wandering Isles
* Stories of Eloran
    * Player Characters and their origins
    * Adventure Starters
    * Artifacts
* Appendices
    * Bestiary and associated tables
    * Suggested Inspirational Media
    * Life path generators
    * Name generator
    * Adventure generator

**Reply by CSLewin** [reply]
Generally excellent. Meet soon to go over details? Some errant demonyms, but mostly to generate concept/task lists for individual chapters/sections. Also to mark out which content is fresh from your head so you can explain it to me--it's possible I've mentioned something in passing and you took it and ran with it, so I'm excited to see where that is.

**Reply by dskiles** [reply]
If you'd like to meet to go over details, I'm usually free after 5pm, or all day Saturdays.

In terms of content that I _think_ is fresh, here's what I know off the top of my head:

* The Event:  I have canonically placed this 27 years ago to mollify my players.
* Mad Pyotr's tower:  This is a Wizard Funhouse Dungeon near Libussa
* The Imperium:  I believe I have filled this in much more than you.  It's highly Rome-inspired, and existed for thousands of years before it collapsed in the aftermath of the Event.
* Dwarves:  They exist
* Elves:  They exist
* Chernik:  Is an authoritarian oligarcho-theocracy that's highly at odds with the cultural mores of both Anem _and_ Pirra.  Also, birds.

We both probably have quite a few other things, but they'll likely surface in each chapter discussion.

**Reply by dskiles** [reply]
OH RIGHT, ORCS AND HALF ORCS.

About 400 years ago, the empire teleported in several thousand orcs from parts unknown to serve as soldiers.  Since then, Orcs have thrived in human society, though they have lost much of their history culture in the process.  Some do their best to integrate, while others hold tightly to what unique identity they still have.

Integrationist orcs are some of the fiercest proponents for reestablishing the Empire, and still hold to many of its cultural customs.  Reclamationist orcs have retreated to the far places of the continent to build something they can call their own.
